experimental/packages/opentelemetry-sdk-node/package.json

```diff
@@ -42,15 +42,15 @@
     "access": "public"
   },
   "dependencies": {
-    "@opentelemetry/api-metrics": "0.26.0",
-    "@opentelemetry/core": "1.0.0",
-    "@opentelemetry/instrumentation": "0.26.0",
-    "@opentelemetry/resource-detector-aws": "0.24.0",
-    "@opentelemetry/resource-detector-gcp": "0.24.0",
-    "@opentelemetry/resources": "1.0.0",
-    "@opentelemetry/sdk-metrics-base": "0.26.0",
-    "@opentelemetry/sdk-trace-base": "1.0.0",
-    "@opentelemetry/sdk-trace-node": "1.0.0"
+    "@opentelemetry/api-metrics": "~0.26.0",
+    "@opentelemetry/core": "~1.0.0",
+    "@opentelemetry/instrumentation": "~0.26.0",
+    "@opentelemetry/resource-detector-aws": "~1.0.0",
+    "@opentelemetry/resource-detector-gcp": "~0.26.0",
+    "@opentelemetry/resources": "~1.0.0",
+    "@opentelemetry/sdk-metrics-base": "~0.26.0",
```

Comment:
I think `~` version ranges on packages `0.x.y` allows `y` to change which could introduce more than just bugfixes. I thought we had agreed to pin these versions for that reason.

Reply:
Yeah. I think we should leave the `0.x` deps pinned.

```shell
> const semver = require('semver')
> semver.satisfies('0.26.1', '~0.26.0')
true
```

Reply:
but `y` should always be allowed to change, not sure what are you referring to ????

Reply:
Allowing the patch part to change when the version is `<1` is the same as allowing the minor part to change when the version is `>=1`

Reply:
> Allowing the patch part to change when the version is `<1` is the same as allowing the minor part to change when the version is `>=1`

I don't you you right, this is all about allowing the version to get patches and that's it nothing more, especially now when package from main requires package from contrib 

https://jubianchi.github.io/semver-check/#/~0.26.0/0.26.3  matches, OK
https://jubianchi.github.io/semver-check/#/~0.26.0/0.27.3, not match, OK
https://jubianchi.github.io/semver-check/#/~0.26.0/1.26.3 not match, OK